source/index.md

```diff
@@ -805,6 +805,17 @@ https://gateway.clearhaus.com/cards/:id
 https://gateway.clearhaus.com/account
 ````
 
+## Changes
+
+### CLP and UGX exponent change
+
+CLP and UGX changes from exponent 2 to exponent 2.
+
+Between 2017-10-12T19:00:00+00:00 and 2017-10-15T19:00:00+00:00 (both inclusive)
+it will not be possible to process transactions in neither CLP nor UGX; before
```

Comment:
Is this double negation? ("not" and "neither") How about "not be possible to ... in CLP and UGX"?

Reply:
I tried an even more direct wording. WDYT?